doc/classes/@GlobalScope.xml

```diff
@@ -1040,6 +1040,26 @@
 				A weak reference to an object is not enough to keep the object alive: when the only remaining references to a referent are weak references, garbage collection is free to destroy the referent and reuse its memory for something else. However, until the object is actually destroyed the weak reference may return the object even if there are no strong references to it.
 			</description>
 		</method>
+		<method name="wrap">
+			<return type="Variant" />
+			<argument index="0" name="value" type="Variant" />
+			<argument index="1" name="min" type="Variant" />
+			<argument index="2" name="max" type="Variant" />
+			<description>
+				Wraps the [Variant] [code]value[/code] between [code]min[/code] and [code]max[/code]. Variant types [int], [float] (real) are supported.
+				If any of the argument is [float] the result will be [float], otherwise it is [int].
```

Comment:
```suggestion
				Wraps the [Variant] [code]value[/code] between [code]min[/code] and [code]max[/code].
				Usable for creating loop-alike behavior or infinite surfaces.
				Variant types [int] and [float] (real) are supported. If any of the argument is [float] the result will be [float], otherwise it is [int].
```
Just a small nitpick: I would add the hint to in which case this function could be useful here as well. (Copied from wrapi/wrapf)